content/en/docs/features/jetstream.md

```diff
@@ -29,6 +29,8 @@ More information about the JetStream can be found [here](https://docs.nats.io/je
 - `ownerID` is the owner of the device. It is calculated as `uuid.NewV5(uuid.NamespaceURL, value of JWT ownerClaim)`.
 - `deviceID` is the UUID of the device.
 - `resourceID` is the unique identifier of resource over the whole hub. It is calculated as `uuid.NewV5(uuid.NamespaceURL, deviceID+href)`, where the `href` is a resource path. (eg "/oic/d").
+- `resourceHrefID` is the identifier of resource href. It is calculated as `uuid.NewV5(uuid.NamespaceURL, resourceHref)`. For multiple hrefs of the same resource, the same `resourceHrefID` is used.
```

Comment:
I wouldn't support subscriptions based on the href. If you have your fleet of same device type connected, let's say 10000 fire alarms, yes, resources are hosted on the same uri, but they have also the same type. I would skip this use-case as what enables interoperability and schema reuse is rt.

Reply:
OK we can drop it now.

Reply:
OK we can drop it now.